WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -180,26 +180,36 @@ fileprivate extension NotificationsTableViewCellContent {
 
 // MARK: - Helpers
 
-private struct SizeCalculator: ViewModifier {
+private struct SizePreferenceKey: PreferenceKey {
```

Comment:
Credits to the author of [this post](https://swiftwithmajid.com/2020/01/15/the-magic-of-view-preferences-in-swiftui/) for this solution